docs/upgrades/v0.4-to-v0.5-upgrade/examples/src/test/kotlin/examples/BreakingChanges.kt

```diff
@@ -10,8 +10,8 @@ import org.partiql.lang.eval.ExprValue
 import org.partiql.lang.eval.ExprValueFactory
```

Comment:
self-review: changes in the other files were to get the build working in the old examples. they shouldn't need modified for future migration guides we add